src/engine/strat_engine/backstore/blockdevmgr.rs

```diff
@@ -345,7 +345,7 @@ impl BlockDevMgr {
             current_time
         };
 
-        let data_size = Bytes(metadata.len() as u64);
+        let data_size = Bytes(metadata.len() as u128);
```

Comment:
We might as well not waste all those ```From``` impls in the devicemapper PR. This could just be ```Bytes::from(metadata.len())``` or some variation thereof. There's a couple other places where I think one of those would make more sense.

Reply:
Okay, yes, this makes sense. I've gone through the entire code base and I think I found everywhere that that could be changed over in the most recent commit.